pkg/alert/alert.go

```diff
@@ -281,19 +281,21 @@ func (s *Sender) Send(ctx context.Context, alerts []*Alert) error {
 	var g errgroup.Group
 
 	for _, u := range s.alertmanagers() {
+		amURL := *u
```

Comment:
we need https://github.com/improbable-eng/thanos/issues/157 someday, let's leave that for later